website/source/docs/provisioning/shell.html.md

```diff
@@ -129,6 +129,33 @@ Vagrant.configure("2") do |config|
 end
 ```
 
+In the code block above, the script block starts with `<<-SCRIPT` and ends with `SCRIPT`.
+This is known as a _Here Document_ or a _heredoc_.  Additionally, if your script
+relies on quotes and you do not wish for Ruby to escape your quotes, you may
+want to use this style of _heredoc_ where `SCRIPT` is surrounded in single quotes:
```

Comment:
I would change _Here Document_ to "here document" and _heredoc_ to "heredoc", and only put the terms in quotes the first time we use them.